packages/unit/test/widget/custom_multi_child_layout_test.dart

```diff
@@ -30,20 +30,32 @@ class TestMultiChildLayoutDelegate extends MultiChildLayoutDelegate {
       performLayoutIsChild = isChild('fred');
     }, returnsNormally);
   }
+
+  bool shouldRelayoutCalled = false;
+  bool shouldRelayoutValue = false;
+  bool shouldRelayout(_) {
+    shouldRelayoutCalled = true;
+    return shouldRelayoutValue;
+  }
 }
 
+Widget buildFrame(MultiChildLayoutDelegate delegate) {
+  return new Center(
+    child: new CustomMultiChildLayout([
+      new LayoutId(id: 0, child: new Container(width: 150.0, height: 100.0)),
```

Comment:
indent the list items and the final ] by two more (if you put the [ on the next line you'll see why)